src/Logger.js

```diff
@@ -58,6 +64,28 @@ class Logger {
     this._log(message);
   }
 
+  verbose(message) {
+    if (this.logLevel < 4) {
+      return;
+    }
+
+    let currDate = new Date();
+    message = `[${currDate.toLocaleTimeString()}]: ${message}`;
+    if (this.logLevel > 4) {
+      if (!this.logFile) {
+        this.logFile = fs.createWriteStream(
+          path.join(
+            process.cwd(),
+            `parcel-debug-${currDate.toLocaleDateString()}@${currDate.toLocaleTimeString()}.log`
+          )
+        );
+        this.logFile.write = this.logFile.write.bind(this.logFile);
```

Comment:
is this bind needed?

Reply:
Don't think so, I'll remove it, seems to work fine without